frontend/static_vol/js/modules/friendList.js

```diff
@@ -31,7 +31,7 @@ const updateFriendsList = async (pageInstance) => {
                 const friend = friends[i];
                 const avatar = friend.avatar ? friend.avatar : '/images/avatar_default.png';
                 //todo: friendsの状況に応じて online/ offline / busy で切り替える
-                const onlineStatus = 'online';
+                const onlineStatus = friend.online ? 'online' : ''; //'online';
```

Comment:
```suggestion
                const onlineStatus = friend.online ? 'online' : 'offline'; //'online';
```
ここがofflineになると嬉しいです！
（avatar右下の、オンラインだと緑色の丸のつくところに灰色の丸がつきます）

Reply:
そうでした！　グレーアウト＆グレーのポチいかにもオフラインでよいです！！